### PR TITLE
Dashboards: Add link to documentation in repeated row warning

### LINKED ISSUE #73930: Add the "Learn More" button once the documentation is done.
Redirect to https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/create-dashboard/#configure-repeating-rows

### PR DESCRIPTION
**Pull request by juanicabanas**
**What is this feature?**

It adds a link to the updated documentation related to the repeated row and the unsupported behavior

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana/issues/73930

**Special notes for your reviewer:**

image

Merge this PR once the documentation is updated and [this other PR](https://github.com/grafana/grafana/pull/73787) is merged.
CC: @imatwawana 

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by dprokop — APPROVED** [review]
Looks good code-wise. I would wait for @imatwawana to review as I'm not 100% sure about the copy of the error message.

**Comment by juanicabanas** [comment]
> Looks good code-wise. I would wait for @imatwawana to review as I'm not 100% sure about the copy of the error message.

The message was decided by @imatwawana and Patrick, so we're ok with that. However, we need to wait until the doc is updated to merge it

**Pull request merged**
